lib/config/migration.spec.ts

```diff
@@ -549,7 +549,7 @@ describe('config/migration', () => {
             packagePatterns: ['^bar'],
             excludePackageNames: ['baz'],
             excludePackagePatterns: ['^baz'],
-            sourceUrlPrefixes: ['https://github.com/vuejs/vue'],
+            sourceUrlPrefixes: ['https://github.com/lodash'],
```

Comment:
Maybe unnecessary 🙃. Will let you guys decide

Reply:
It's fine